cypress/e2e/shoppingBag_spec.cy.js

```diff
@@ -0,0 +1,69 @@
+describe(('Shopping Bag Page'), () => {
+  beforeEach(() => {
+    cy.intercept('POST', 'https://everuse-be-b6017dbfcc94.herokuapp.com/', {
+      statusCode: 201,
+      fixture: 'mockBag'
+    }).as('getAllItems')
+    cy.visit('http://localhost:3000/')
+    cy.get('a').click();
+  })
+  
+  it('Should show shopping bag details', () =>{
+    cy.wait('@getAllItems').then((interception) => {
+      cy.get('h1').contains('EverUse');
+      cy.get('h2').contains('Shopping Bag');
+      cy.get('.bag__items').children().should('have.length', 3);
+      cy.get('.bag__items').children().first()
+        .get('.item__title').contains('bracelet')
```

Comment:
Alot of cy.gets with .gets peppered in. Could clean up a little.

Reply:
I try to organize them per assertion, so if I'm testing everything in the first child, I will do .gets and only put a semicolon at the very end. But if that isn't translating, then I guess it isn't a very helpful. method lol